Sources/SwagGenKit/CodeFormatter.swift

```diff
@@ -80,11 +80,15 @@ public class CodeFormatter {
         context["filename"] = getFilename(schema.name)
         context["type"] = getModelType(schema.name)
 
-        if case .simple = schema.value.type {
+        switch schema.value.type {
+        case .simple:
             context["simpleType"] = getSchemaType(name: schema.name, schema: schema.value)
             if let enumValue = schema.value.getEnum(name: schema.name, description: schema.value.metadata.description) {
                 context["enum"] = getEnumContext(enumValue)
             }
+        case .reference:
+            context["refType"] = getSchemaType(name: schema.name, schema: schema.value)
```

Comment:
Can we rename this `referenceType`? Do you think it's actually worth splitting into a seperate property? It could just use `simpleType`, but I guess some template languages may want to generate this differently?

Reply:
I made this some time ago so I do not recall exactly why I used a different key 🤔
Because, looking back to your templates and mine (I use fully hand-made stencils), I do not take advantage of this information.
Maybe we could do both so that the information is not lost: assigning `simpleType` and `referenceType`?

Note that after this PR will come [array support](https://github.com/Liquidsoul/SwagGen/commit/976dde952a5be0da42c8f52aca611825e3deeee4) which could be solve the same way (`simpleType` and `arrayType`).

Reply:
Yeah I'm happy with that. Adding the specific context, but perhaps always adding `aliasType` as well, that the template can use

Reply:
You want to use `aliasType` instead of `simpleType`?

Reply:
Yeah. We can keep `simpleType` in the context, but add `aliasType`  (as well as to the new reference and array types). Then the template only has to reference `aliasType` instead of 3 different properties

Reply:
I've updated the PR, tell me if its ok for you. Do you want me to add the `arrayType` commit as well in this PR or do you want to treat it separately?